app/models/rubric_criterion.rb

```diff
@@ -12,6 +12,8 @@ class RubricCriterion < Criterion
 
   has_many :tas, through: :criterion_ta_associations
 
+  has_many :levels
```

Comment:
This is good; add a default order scope on the level number (see https://stackoverflow.com/questions/18284606/deprecated-warning-for-rails-4-has-many-with-order for details).

Reply:
should I order on level number or level mark?